imap/message.c

```diff
@@ -738,21 +738,20 @@ message_header_lookup(const char *buf, const char **valp)
 static void body_add_content_guid(const char *base, struct body *body)
 {
     int encoding = ENCODING_NONE;
-    char *decbuf = NULL;
+    uint8_t dest[20];
```

Comment:
`xsha1.h` already defines the macro `SHA1_DIGEST_LENGTH`, prefer to use it here instead of magic `20`?

Reply:
Yep, updated here and in the cunit tests too